pgqueuer/qm.py

```diff
@@ -94,22 +97,55 @@ def __init__(
         self,
         func: Entrypoint,
         requests_per_second: float,
+        heartbeat_rate: timedelta | None,
     ) -> None:
         self.func = func
         self.requests_per_second = requests_per_second
         self.is_async = is_async_callable(func)
+        self.heartbeat_handle: asyncio.TimerHandle | None = None
+        self.heartbeat_rate = heartbeat_rate
```

Comment:
heartbeat_rate should consider retry-timer given by run as well, given by entrypoint should override given by run(...)